src/test/kotlin/org/web3j/OpenethereumGreeterTest.kt

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2019 Web3 Labs Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package org.web3j
+
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Test
+import org.web3j.greeter.Greeter
+import org.web3j.protocol.Web3j
+import org.web3j.protocol.core.DefaultBlockParameterName
+import org.web3j.tx.TransactionManager
+import org.web3j.tx.gas.ContractGasProvider
+
+@EVMTest(NodeType.PARITY)
```

Comment:
I would also rename the `NodeType.PARITY` to `NodeType.OPEN_ETHEREUM`.

It was not supported before anyway so nobody is using it.